porting/first-steps.rst

```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc lzop imagemagick libncurses5 rsync
 
+.. Note::
+    repo is not available in repositories of Ubuntu 20.04 or later versions. Hence it has to be installed manually.
+
+
+Installing repo 
+^^^^^^^^^^^^^^^^^
+
+To install repo, we need to ensure that you curl and git is installed. To install curl & git run the following command :
+
+.. code-block::
```

Comment:
Did you see that travis CI check has failed? i think this line should be

.. code-block:: shell

Reply:
Corrected as part of next commit.